README.md

```diff
@@ -31,6 +31,97 @@ GetText/DecorateFunctionMessage:
   Enabled: true
 ```
 
+## Cops
+### GetText/DecorateFunctionMessage
+This cop looks for any raise / fail functions and checks the user visible message is using gettext decoration with the _() function.
```

Comment:
Suggest replacing 'raise / fail' with 'raise or fail'

Also "checks the user visible message is using ..." to "checks that the user message is using ..."